publication-event-dtopublicationdoi-producer/src/test/resources/dynamodbevent_new_only.json

```diff
@@ -504,7 +504,7 @@
           "bs": null
         },
         "status": {
-          "s": "Topppublisering",
+          "s": "Draft",
```

Comment:
Do we want to maybe save enum values as CAPITAL letters?

Reply:
See https://github.com/BIBSYSDEV/nva-publication-api/pull/95/files#r522086817

Reply:
Question for database guru @axthosarouris

Reply:
I think we either want to save them in Capital or as URIs. I discussed this with @brinxmat but we didn't arrive on any actions yet.

Reply:
> Question for database guru @axthosarouris

I agree that we should save enum values with capital letters.   I think however, that when we read a value we should be tollerant with case. We write using capitals, we accept aNY cASe.

 URIs are probably a better idea, the only problem, being that we should have an endpoint for dereferencing them which now we don't. If we find it necessary to use URIs as enums, then we must have a plan for documenting and maintaining the dereferencing pages of these constants. 

My suggestion is that in the scope of this PR, we should use capital letters. When we have established a way to collect and document all exposed enums, then the transition will not be a problem as we can allow the enums to accept both strings and URIs and output only URIs.

P.S.
I am definitely not a guru :)

Reply:
updated